invenio_app_ils/ui/backoffice/src/App.test.js

```diff
@@ -1,9 +1,9 @@
-import React from "react";
-import ReactDOM from "react-dom";
-import App from "./App";
+import React from 'react';
```

Comment:
Are we using Prettier? because, for me, it is putting this `"` in the import...
We need to all align on this.

Reply:
Yes we use it! I had the same issue but I don't remember how I fixed it.

Reply:
But did you put any config file anywhere?